src/dcs/core/data_repository.py

```diff
@@ -81,7 +81,8 @@ def check_server_uri(cls, value: str):
         """Checks the drs_server_uri."""
         if not re.match(r"^drs://.+/$", value):
             raise ValueError(
-                f"The drs_server_uri has to start with 'drs://' and end with '/', got : {value}"
+                f"The drs_server_uri has to start with 'drs://' and end with '/', got : {
+                    value}"
```

Comment:
Now I'm assuming ruff did this

Reply:
I think in Python 3.12 f-string expressions can now be multi-line, so probably this is ok.

Reply:
But maybe better to keep the lines shorter, so that ruff doesn't try to break them:
```python
       "The drs_server_uri has to start with 'drs://' and end with '/'"
       + f", got : {value}"
```

Reply:
Also, consider this: https://docs.astral.sh/ruff/rules/f-string-in-exception/

Reply:
Thanks for the pointer, that's generally inconsistent across the services and could be changed oportunistically.